sdk/base-sys/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "miden-base-sys"
+description = "Miden rollup Rust bingings and MASM library"
+version = "0.0.0"
+rust-version.workspace = true
+authors.workspace = true
+repository.workspace = true
+homepage.workspace = true
+documentation.workspace = true
+categories.workspace = true
+keywords.workspace = true
+license.workspace = true
+readme.workspace = true
+edition.workspace = true
+
+[dependencies]
+miden-assembly.workspace = true
+miden-stdlib-sys = { version = "0.0.1", path = "../stdlib-sys", optional = true }
+
+[build-dependencies]
+miden-assembly.workspace = true
+
+[features]
+default = []
+
+# User facing Rust bindings
+"bindings" = ["miden-stdlib-sys"]
```

Comment:
I think you can simply make the `masl-lib` parts of this use `#[cfg(not(target_family = "wasm"))]`, rather than a Cargo feature. This keeps it out of the Wasm builds that would cause this to end up in code processed by the compiler, but otherwise keeps all of the code together.

In the future we may need to add a feature that allows those items to exist in a Wasm build as well (for building the compiler to run in the browser), but we're a way away from that point yet.

Reply:
Or vice versa, make the bindings conditional on `#[cfg(target_family = "wasm")]` - that might make more sense, since the bindings only are needed when targeting Miden.